components/atom/upload/package.json

```diff
@@ -11,7 +11,6 @@
   "dependencies": {
     "@s-ui/component-dependencies": "1",
     "@s-ui/react-atom-button": "1",
-    "@s-ui/react-atom-icon": "1",
```

Comment:
this should be a major release but I can't find the proper commit message

Reply:
done!